individual/workflows/base_individual_update.py

```diff
@@ -80,9 +80,15 @@ def process_update_individuals_workflow(user_uuid, upload_uuid):
             SET first_name = COALESCE(f."Json_ext"->>'first_name', first_name),
             last_name = COALESCE(f."Json_ext"->>'last_name', last_name),
             dob = COALESCE(to_date(f."Json_ext"->>'dob', 'YYYY-MM-DD'), dob),
+            location_id = loc."LocationId",
             "DateUpdated" = NOW(),
             "Json_ext" = f."Json_ext"
             FROM individual_individualdatasource f
+            LEFT JOIN "tblLocations" AS loc
+                    ON loc."LocationName" = f."Json_ext"->>'location_name'
+                    AND loc."LocationCode" = f."Json_ext"->>'location_code'
```

Comment:
I will merge it but location code is unique therefore enough, having both the code and the name might create rejection if the name has a typo in it (or written in an other language)

Reply:
Fair consideration. We discussed internally the option to include only location code but in the ended decided to go with both because location code uniqueness is not guaranteed/required at the system level. Additionally, with name and code both present it's easier for troubleshooting when an individual record is rejected due to typo in either field – if it were code only, it would be more challenging for users to identify the originally intended location.